docs/getting-started/sdk/secrets-manager/integrations/kubernetes.mdx

```diff
@@ -0,0 +1,300 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Kubernetes
+
+The Bitwarden Secrets Manager Kubernetes Operator (sm-operator) is a tool for teams to integrate
+Bitwarden Secrets Manager into their Kubernetes workflows seamlessly.
+
+The sm-operator uses a [controller](internal/controller/bitwardensecret_controller.go) to
```

Comment:
@jhbeskow The build action is complaining that `internal/controller/bitwardensecret_controller.go` is a broken link:
<img width="909" alt="Screenshot 2024-04-04 at 10 04 23 AM" src="https://github.com/bitwarden/contributing-docs/assets/8926729/9f13bfe7-805d-4af9-9bb3-a41e982a5f8e">

Reply:
I missed that one.  I will get it.